src/lib.rs

```diff
@@ -66,5 +66,7 @@ pub mod fallback_settings;
 mod install;
 mod notifications;
 pub mod settings;
+// #[cfg(test)]
```

Comment:
?

Reply:
oh hmm, I think I wanted that to be cfg(test) but it can't be for the integration tests to use the test helpers, they need to be built normally and then dropped during link